enginetest/join_planning_tests.go

```diff
@@ -55,7 +55,10 @@ var JoinPlanningTests = []struct {
 			"insert into rs values (0,0), (1,0), (2,0), (4,4), (5,4);",
 			"insert into uv values (0,1), (1,1), (2,2), (3,2);",
 			"insert into ab values (0,2), (1,2), (2,2), (3,1);",
-			"update information_schema.statistics set cardinality = 1000 where table_name in ('ab', 'rs', 'xy', 'uv');",
+			`analyze table xy update histogram on x using '{"row_count":1000}'`,
```

Comment:
Need to test the error case: giving a non-json data value, or one that doesn't have the right fields